src/lib.rs

```diff
@@ -1,4 +1,4 @@
-mod bus;
+pub mod bus;
```

Comment:
I think the `bus` module can stay private since we're already exporting `bus::Bus` in the same file.

https://github.com/matrix-io/matrix-rhal/blob/e6469e88e4832ce67b62664f30b49324c75c6719/src/lib.rs#L7

Reply:
I think it's needed to get at `matrix_rhal::bus::memory_map`